package.json

```diff
@@ -22,15 +22,18 @@
   "dependencies": {
     "eslint-plugin-node": "11.1.0",
     "eslint-plugin-security": "1.4.0",
-    "eslint-plugin-react": "7.24.0"
+    "eslint-plugin-react": "7.33.0"
   },
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^3.7.0",
     "@typescript-eslint/parser": "^3.7.0",
-    "eslint": "^7.32.0",
+    "eslint": "^7.32.0 || ^8",
```

Comment:
Wasn't sure whether to add v8 here or whether to just specify ^8.45.0 and replace this declaration

Reply:
The ESLint version here is used for testing the rules, rather than being exposed as the "supported" version of ESLint for the plugin.

Since this change would mean that the version of ESLint used by the repo could jump a major version and break the tests, I'd advise leaving it where it is - updating the rules and their tests should be a separate issue to upgrading the exposed dependencies :)